lib/mutant/reporter/cli/printer.rb

```diff
@@ -3,10 +3,34 @@ class Reporter
     class CLI
       # CLI runner status printer base class
       class Printer
-        include AbstractType, Delegator, Adamantium::Flat, Concord.new(:output, :object), Procto.call(:run)
+        include AbstractType,
+                Adamantium::Flat,
+                Concord.new(:output, :object),
+                Procto.call(:run)
 
         private_class_method :new
 
+        # Create delegators to object
+        #
+        # @return [undefined]
+        def self.delegate(*names)
```

Comment:
Are there other places that use delegation within the codebase? If so should this be extracted into a module and used here?

Even if it's not used anywhere else, this delegation plumbing code might be better off separated from the system domain.